minio/minio
# Minio start permission issue

Fixes #9746
When there is an ownership problem with the data directory or endpoint during minio server start time, the provided HINT in the error message was not applied to the internal `.minio.sys` directory and its content since "-R" option was not included in the suggestion.

## Motivation and Context
To give user the correct HINT that works

## How to test this PR?
- Start minio server with a simple data directory, like `/tmp/dir/` as user "user1"
- Change user and be "user2" and run the same command to start minio server at the same data directory
- You'll hit the error message with the HINT. Run the proposed first command in the error message to fix the error conditon and make sure the same command can start the minio server 

## Types of changes

## Checklist:
